src/main/java/core/basesyntax/impl/ParseDataServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.servise.ParseDataService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParseDataServiceImpl implements ParseDataService {
+    @Override
+    public List<FruitTransaction> parse(List<String> dataFromFile) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (int i = 0; i < dataFromFile.size(); i++) {
+            String line = dataFromFile.get(i);
+            String [] records = line.split(",");
+            transactions.add(new FruitTransaction(getOperation(records[0]),
+                    new Fruit(records[1]), Integer.parseInt(records[2])));
+        }
+        return transactions;
+    }
```

Comment:
Are you sure we don't need validate input data? 
Use constants instead of numbers when referring to array elements.